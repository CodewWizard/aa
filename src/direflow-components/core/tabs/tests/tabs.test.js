import React from 'react';
import renderer from 'react-test-renderer';
import TabsGroup from '../tabs'
import TestRenderer from 'react-test-renderer';

const AlertArgs = {
  type: "alert",
  size: "md",
  successLabel: "OK",
  header: "Alert!",
  showModal: true,
  message: "Are you sure?",
  onSuccess:() => {},
  onClose: () => {}
}

it('renders without crashing', () => {
    const testRenderer = TestRenderer.create(<TabsGroup {...AlertArgs} />);
    testRenderer.unmount()
  });

  it('matches snapshot as expected', () => {
    const renderTree = renderer.create(<TabsGroup {...AlertArgs} />);
    expect(renderTree).toMatchSnapshot();
  });