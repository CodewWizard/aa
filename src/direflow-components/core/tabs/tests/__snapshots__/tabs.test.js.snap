// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot as expected 1`] = `
Array [
  <nav
    className="mb-3 compaTabs nav nav-tabs"
    onKeyDown={[Function]}
    role="tablist"
  />,
  <div
    className="tab-content"
  />,
]
`;
